src/objects.rs

```diff
@@ -449,6 +465,7 @@ impl ObjectData {
             kerning,
             halign,
             valign,
+            contents,
```

Comment:
I wonder if we could just call this `text`? I realize this may be stored without explicit name in the XML format, but `text` would match the property name used in the JSON format: https://doc.mapeditor.org/en/stable/reference/json-map-format/#text